app/views/_user_util_links.html.erb

```diff
@@ -67,15 +67,16 @@ Unless required by applicable law or agreed to in writing, software distributed
           <% end %>
           <% end %>
         </ul>
-    </div>
+      </div>
+    </li>
   <% end %>
 
   <li class="divider desktop-hidden" />
   <%= link_to_if user_signed_in?, 'Sign out', main_app.destroy_user_session_path do %>
   <%# Fallback if the test above fails %>
   <%= link_to 'Sign in', main_app.new_user_session_path %>
   <% end %>
-  <% unless current_page?(main_app.root_path) %>
+  <% unless current_page?(root_path) %>
```

Comment:
@Dananji Was there a reason why you reverted this from `main_app.root_path` to `root_path`?

Reply:
Sorry, I must have changed it accidentally. The only change I wanted to do was put the `Manage` menu and button within a `<li>` element. I will revert this back.